### PR TITLE
add additional documentation for the ray tracing class

### PR DESCRIPTION
**Pull request by cg-laser**

### CODE REVIEW

**Review by fschlueter — DISMISSED** [review]

File: NuRadioMC/SignalProp/analyticraytracing.py [hunk 1]
```diff
@@ -2683,6 +2683,23 @@ def get_output_parameters(self):
...
+            Index of the raytracing solution
+
+        Returns
+        -------
+        output_dict: dict
+            Dictionary containing the output of the ray tracing
+            The C_0 and C_1 parameters are the parameters of the analytic function that describes the ray path
+            The solution type is the type of the solution (1: direct, 2:refracted, 3: reflected off the surface)
+            The reflection parameter specifies the number of bottom reflections (in case of reflective layers in the ice)
+            The reflection case specifies if the ray starts upward or downward (1: upward, 2: downward) (only relevant if bottom reflection > 0)
```
Comment (sjoerd-bouma) [inline NuRadioMC/SignalProp/analyticraytracing.py:2701]: Some full stops (`.`) would help with readability, especially online as single line breaks are not preserved in .rst.

Comment (cg-laser) [reply NuRadioMC/SignalProp/analyticraytracing.py]: done

**Review by sjoerd-bouma — APPROVED** [review]
Thanks Christian!

**Review by colemanalan — APPROVED** [review]

**Comment by fschlueter** [comment]
@cg-laser can we merge this?

**Pull request merged**